spec/features/tag_importer_spec.rb

```diff
@@ -187,10 +187,8 @@ def then_i_see_the_import_failed
     row = first('table tbody tr')
 
     expect(row).to have_selector('.label-danger', text: state)
-    expect(row).to have_selector('.label-danger[data-toggle="tooltip"]')
-    expect(row).to have_selector(
-      ".label-danger[data-original-title='#{tagging_spreadsheet.error_message}']"
-    )
+    expect(row).to have_selector('span.error-message', text: tagging_spreadsheet.error_message)
```

Comment:
We could look for only the class name here (e.g. `have_selector('.error-message'`). If we change the `span` to something else, the tests would still be green.